src/Microsoft.Extensions.DependencyInjection/ServiceLookup/IServiceCallSite.cs

```diff
@@ -1,17 +1,12 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.Linq.Expressions;
-
 namespace Microsoft.Extensions.DependencyInjection.ServiceLookup
 {
     /// <summary>
     /// Summary description for IServiceCallSite
     /// </summary>
     internal interface IServiceCallSite
```

Comment:
Marker interface? Am I correct in assuming you could change all instances of `IServiceCallSite` to `object` and the code would still compile and work?